code/modules/mod/mod_types.dm

```diff
@@ -495,7 +495,7 @@
 
 /obj/item/mod/control/pre_equipped/responsory/janitor
 	insignia_type = /obj/item/mod/module/insignia/janitor
-	additional_module = /obj/item/mod/module/noslip
+	additional_module = list(/obj/item/mod/module/noslip, /obj/item/mod/module/mister/cleaner)
```

Comment:
does this support being a list? if yes you should change it to be called `additional_modules`